joblib/test/test_func_inspect.py

```diff
@@ -207,13 +213,13 @@ def test_clean_win_chars():
         assert char not in mangled_string
 
 
-def test_format_signature():
+@parametrize(['func', 'args', 'kwargs', 'sgn_expected'],
+             [(g, [list(range(5))], {}, 'g([0, 1, 2, 3, 4])'),
+              (k, [1, 2, (3, 4)], {'y': True}, 'k(1, 2, (3, 4), y=True)')])
```

Comment:
Your changes are fine, but remember what I said above about not changing unrelated things if there is not a strong motivation behind it.